README.md

```diff
@@ -47,4 +47,16 @@ yarn cypress:dev  # to develop tests
 
 ```
 yarn cypress:run  # to run all tests
-```
\ No newline at end of file
+```
+
+## Env files
+
+By default app will try to load .env.$NODE_ENV file and fallback to .env. You can use ENV env variable to override this mechanism for example: ENV=dev will load .env.dev file.
```

Comment:
here we miss docs about local file and using secret to override grapgql configuration

Reply:
> local file
The original cherry-picked commit did not contain that information either.
> using secret to override grapgql configuration
Again, should we announce this secret?

Reply:
We shouldn't announce the secret value but the mechanism itself is public.

Reply:
Why do we want to make clear what this mechanism is about?

Reply:
Parameter included in the docs